### PR TITLE
Attempt at terminal tips xD

### PR DESCRIPTION
**Pull request by metonymic-smokey**
Just a draft of a few tips I wanted to share.
Review plis for changes and suggestions xD
I'll add more tips along the same lines if these are okay.

Thanks!

### CODE REVIEW

**Comment by metonymic-smokey** [comment]
@tfidfwastaken since this PR is a little messy with so many commits, do you want me to rebase and squash all these into 1?

**Comment by tfidfwastaken** [comment]
@metonymic-smokey No need to squash and rebase, a slightly messy commit history is not a big deal for us

File: _posts/2018-09-08-terminal-tips.md [hunk 1]
```diff
@@ -3,4 +3,27 @@ title: Terminal tips from PES Open Source
 tags: linux, bash, compilation
 ---
 
-// Start your contributions here
+*Contributed by: Aditi*		      
+## Short tricks for long commands:     
+- Use `Ctrl+A` and `Ctrl+E` to navigate to the beginning and end of the line respectively.   
+- Hit `Ctrl+R` and enter a word/phrase from a previously used super long command and it will retrieve the last used command with the term.  
+
+## Reusing previous commands
```
Comment (tfidfwastaken) [inline _posts/2018-09-08-terminal-tips.md:21]: This section isn't 100% clear to me
> reusing the previous command with the current one

What exactly does 'one' refer to here? Reuse previous command with the current ?command¿ 
I have the same confusion with your `?!` tip.

Maybe you can improve the description a bit and add an example for `?!` to make it more clear?

File: _posts/2018-09-08-terminal-tips.md [hunk 2]
```diff
@@ -3,4 +3,27 @@ title: Terminal tips from PES Open Source
...
+- `!!` is handy for reusing the previous command with the current one.  
+- `!?` uses the argument of the previous command with the current one.
+
+## Bash history
+To avoid having a command added to your .bash_history file, prefix it with a space i.e. go with `  [command]` instead of `[command]`.
```
Comment (tfidfwastaken) [inline _posts/2018-09-08-terminal-tips.md]: Also while you are making the above change, markdown seems to not be adding in the space for ` [command]`
I think you'll have to use`<code>&nbsp;command</code>` instead to make it work

**Review by tfidfwastaken — CHANGES_REQUESTED** [review]
Great additions!
Just make the minor changes I requested and we'll get it merged soon

File: _posts/2018-09-08-terminal-tips.md (hunk 1 above)
Comment (metonymic-smokey) [reply _posts/2018-09-08-terminal-tips.md:21]: @tfidfwastaken resolved the !! tip.
Will add the ?! tip later.
Is it okay now?

Thanks!

Comment (tfidfwastaken) [reply _posts/2018-09-08-terminal-tips.md:21]: Yeah looks good
I added an example for more clarity
LGTM

**Review by tfidfwastaken — APPROVED** [review]

**Pull request merged**